### PR TITLE
[WIP] Expand Software Offering: XMind

### PR DESCRIPTION
**Pull request by 89luca89**
Add XMind Application

This PR is part of a rework on another PR
I splitted in many little PR as requested

### CODE REVIEW

File: plugins/xmind.plugin/install.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+URL="http://dl2.xmind.net/xmind-downloads/xmind-8-update3-linux.zip"
+wget -c "$URL" -O "xmind.zip"
+if [[ ! -f "xmind.zip" ]]; then
+	exit 1
+fi
+rm -rf "/opt/xmind"
+mkdir -p "/opt/xmind"
+unzip -xq "xmind.zip" -d "/opt/xmind"
+
+if (( $? == 0));then
```
Comment (kwizart) [inline plugins/xmind.plugin/install.sh]: I don't think this test is accurate to discover the right arch

File: plugins/xmind.plugin/install.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
+CACHEDIR="/var/cache/fedy/xmind";
+
+mkdir -p "$CACHEDIR"
+cd "$CACHEDIR"
+
+arch | grep x86_64
```
Comment (kwizart) [inline plugins/xmind.plugin/install.sh]: what's the point of that here ? you aren't making use of the test.

Comment (89luca89) [reply plugins/xmind.plugin/install.sh]: Yeah I will fix this, It's not working this way

Should work on another method to pick the right arch?

**Comment by bddenhartog** [comment]
Closing - rebase necessary and I was unable to push LFS data to the original author's repository. See #580.

**Pull request closed without merging**